subsys/bluetooth/controller/Kconfig

```diff
@@ -524,6 +525,11 @@ config BT_CTLR_ASSERT_HANDLER
 	  and will be invoked whenever the controller code encounters
 	  an unrecoverable error.
 
+config BT_CTLR_DF
```

Comment:
`BT_CTLR_DF` should not be visible if not supported

Reply:
Fixed, it was caused by leftover from main controller Kconfig where second BT_CTLR_DF was defined.